src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -264,10 +270,40 @@ public async ValueTask<DbConnectorResultSets> QueryMultipleAsync(CancellationTok
 			return new DbConnectorResultSets(command, await methods.ExecuteReaderAsync(CachedCommand.Unwrap(command), cancellationToken).ConfigureAwait(false), methods);
 		}
 
+		/// <summary>
+		/// Sets the command's timeout.
+		/// <remarks>Throws <c><see cref="ArgumentOutOfRangeException"/></c> if <c>timeSpan</c> is not positive or <c><see cref="Timeout.InfiniteTimeSpan"/></c>.</remarks>
```

Comment:
There is an `<exception>` element we could use here; either way, it should go after `</summary>`.